gulpfile.js

```diff
@@ -65,20 +75,42 @@ const rollupPackages = {
 
 const modules = Object.keys(packages);
 const rollupModules = Object.keys(rollupPackages);
+const allModules = [...modules, ...rollupModules];
+
 const distId = process.argv.indexOf("--dist");
 const dist = distId < 0 ? sources : process.argv[distId + 1];
 
-rollupModules.concat(modules).forEach(mod => {
-  gulp.task(`${mod}:install`, async () => {
-    const packageName = path.resolve(__dirname, `${sources}/${mod}`);
-    await install(packageName);
-  });
-});
+const install = async () => {
+  return Promise.all(
+    allModules.map(mod => {
```

Comment:
`allModules.map(mod => installPackage(path.resolve(__dirname, `${sources}/${mod}`)))`

Reply:
existing solution is more readable, those gulpfiles get really nasty very fast so I'd rather be more verbose